prov/rxm/src/rxm.h

```diff
@@ -299,8 +302,18 @@ struct rxm_rndv_hdr {
 	uint8_t count;
 };
 
-#define rxm_pkt_rndv_data(rxm_pkt) \
-	((rxm_pkt)->data + sizeof(struct rxm_rndv_hdr))
+struct rxm_rndv_raw_key_hdr {
+	uint64_t base_addr;
+	uint8_t raw_key[];
+};
```

Comment:
This probably requires a raw_key that's a multiple of 64-bits, or base_addr will be mis-aligned.

Something looks missing here to me.  I would expect the existing rendezvous headers to carry the address and key of the memory region to read.  But I don't see that there are changes to the existing headers.

Reading down, it looks like this header is supposed to follow struct rxm_rndv_hdr, but at least key information from the rxm_rndv_hdr would be ignored.

I wonder if it wouldn't be cleaner to have 2 rendezvous headers with the selection based on MR_RAW being needed.